virtcontainers/kata_agent.go

```diff
@@ -142,6 +144,17 @@ var kataHostSharedDir = func() string {
 	return defaultKataHostSharedDir
 }
 
+func getPagesizeFromOpt(fsOpts []string) string {
+	//example options array: "rw", "relatime", "seclabel", "pagesize=2M"
+
+	for _, opt := range fsOpts {
+		if strings.Contains(opt, "pagesize") {
```

Comment:
This is still sloppy, since it will get confused by an option which contains the works "pagesize" but not at the beginning.  I really think we should do this properly: check explicitly for a prefix of "pagesize=" and nothing else.

Reply:
sure, I'll change it